Sources_v3/ChatClient.swift

```diff
@@ -54,9 +54,7 @@ public class Client<ExtraData: ExtraDataTypes> {
     private(set) var backgroundWorkers: [Worker]!
             
     /// The notification center used to send and receive notifications about incoming events.
-    private(set) lazy var eventNotificationCenter = environment.notificationCenterBuilder([
-        EventDataProcessorMiddleware<ExtraData>(database: databaseContainer)
-    ])
+    @Cached private var eventNotificationCenter: EventNotificationCenter
```

Comment:
We can't do this. The instance must stay the same. Other workers and elements in the system can use it.

Reply:
I think you can make the `excludedUserIds` variable computed or something like that, to make this work without having to recreate the center.